react-components/tests/tests-utilities/fixtures/wrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { createRenderTargetMock } from './renderTarget';
+import { sdkMock } from './sdk';
+import { SDKProvider } from '../../../src/components/RevealCanvas/SDKProvider';
+import { ViewerContextProvider } from '../../../src/components/RevealCanvas/ViewerContext';
+
+const queryClient = new QueryClient();
+const renderTargetMock = createRenderTargetMock();
+
+export const wrapper = ({ children }: { children: any }): any => (
```

Comment:
Don't use `any` here. Please use correct type, e.g.
```suggestion
export const wrapper = ({ children }: React.PropsWithChildren): JSX.Element => (
```